src/sdam/monitor.ts

```diff
@@ -99,12 +100,16 @@ export class Monitor extends TypedEventEmitter<MonitorEvents> {
   /** @internal */
   [kMonitorId]?: MonitorInterval;
   rttPinger?: RTTPinger;
+  /** @internal */
+  override mongoLogger: MongoLogger;
```

Comment:
We should not need this override correct? it should be inherited from the parent class